src/sentry/models/sentryappinstallation.py

```diff
@@ -9,6 +9,10 @@
 from sentry.db.models import (FlexibleForeignKey, ParanoidModel)
 
 
+def default_uuid():
+    return six.binary_type(uuid.uuid4())
```

Comment:
You have this function in a couple places. Could it be hoisted into `sentry.db.models` or `sentry.models`?

Reply:
Its only this way because in Django 1.8 you have to have a canonical (importable) reference to the function. I'd consider these functions local to the model.

Reply:
Ok :+1: